example.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # Copyright (c) 2018 Sam Wenke (samwenke@gmail.com)
+# Copyright (c) 2019 Ingvar Lond (ingvar.lond@gmail.com)
```

Comment:
It'd be useful to add this to player_features in env.py and also the amount the total that the player has bet for this hand:

int(player.hand_starting_stack- player.stack)

```
      player_features = [
        int(player.emptyplayer),
        int(player.get_seat()),
        int(player.stack),
        int(player.playing_hand),
        int(player.handrank),
        int(player.playedthisround),
        int(player.betting),
        int(player.isallin),
        int(player.lastsidepot),
        int(player.hand_starting_stack- player.stack),
        int(player.hand_starting_stack),
      ]
```

Reply:
Not the objective of this batch of work.

Reply:
By the way, what you are trying to do here is feature engineering and this should be done outside the environment. As should "handrank" be removed from environment output. That is not part of the state, but something a ANN should learn (or you add it manually when encoding the state)